app/services/workload-points-service.js

```diff
@@ -14,16 +15,27 @@ module.exports.getWorkloadPoints = function (id, organisationLevel) {
     new Link('Admin', '/admin')
   ]
 
-  return getWorkloadPoints().then(function (results) {
-    if (results !== undefined) {
-      var formattedUpdateDate = dateFormatter.formatDate(results.effectiveFrom, 'DD/MM/YYYY')
-      results.effectiveFrom = formattedUpdateDate
+  return getWorkloadPoints().then(function (workloadPoints) {
+    if (workloadPoints !== undefined) {
+      var formattedUpdateDate = dateFormatter.formatDate(workloadPoints.effectiveFrom, 'DD/MM/YYYY')
+      workloadPoints.effectiveFrom = formattedUpdateDate
     }
-    result.title = breadcrumbs[0].title
-    result.subTitle = breadcrumbs[1].title
-    result.workloadPoints = results
-    result.breadcrumbs = breadcrumbs
-    return result
+    return userRoleService.getUserById(workloadPoints.updatedByUserId)
+    .then(function (user) {
+      var updatedBy = workloadPoints.updatedByUserId // Default to the user id
+      if (user) {
+        if (user.name) { // If there is a valid use that
+          updatedBy = user.name
+        }
+        updatedBy = user.username
```

Comment:
There needs to be an else here or the name will always get over written by the username.